src/components/Game.js

```diff
@@ -25,12 +64,15 @@ const Game = () => {
         { exampleFormat }
       </p>
 
-      <RecentSubmission />
+      <div className={showRecentSubmission() ? 'visible' : 'hidden'}>
+        <RecentSubmission submission={poem[poem.length-1]}/>
```

Comment:
Just note that if `poem` is empty this will send `undefined` to the prop `submission` and that's why you're getting a warning.